build/Dockerfile.tracerunner

```diff
@@ -14,7 +14,7 @@ RUN make _output/bin/trace-runner
 FROM ubuntu:19.10
 
 RUN apt-get update
-RUN apt-get install -y xz-utils
+RUN apt-get install -y kmod xz-utils bpfcc-tools
```

Comment:
is this what the other PR did? Would be better to install the BCC tools directly, but i think we need a patch into bcc to generate the packages properly

Reply:
Yes this is similar to what the other patch was doing.

>  Would be better to install the BCC tools directly
Does this mean updating the `iovisor/bpftrace` image to include BCC tools directly? If yes, I can open up a PR to do that.